CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### 2.1 
+ * IMPORTANT UPGRADE NOTE FOR CRUNCHY PACKAGE USERS: In version 2.0, the Crunchy provided extras for node_exporter were split out from the pg##-extras package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or less, if you have node_exporter and postgres_exporter running on the same systems, ensure that you install the separate node_exporters_extras package after the update. See the README for the full package name(s).
+
+ * Fixed Grafana data source to use the "proxy" mode instead of "direct" with default install. Should fix issues encountered with the default setup when Grafana is not installed on the same system as Prometheus.
+ * Renamed functions_pg##.sql file to setup_pg##.sql to better clarify what it's for (and because it's not just functions).
+ * Added ccp_wal_activity metric to help monitor WAL generation rate. Note that for PG96 and lower, an new security definer function must be added (can just run setup_pg##.sql again). New metric definition is located in the queries_pg##.yml file. No default rules have been added since this is very use-case dependent.
+ * Improved accuracy of Idle In Transaction monitoring times in PostgreSQL. Base the time measured on the state change of the session vs the total transaction runtime.
```

Comment:
* Improved accuracy of "Idle In Transaction" monitoring times in PostgreSQL. It is now based on the time measured on the state change of the session vs the total transaction runtime.